core/templates/dev/head/components/ck-editor-helpers/ck-editor-4-widgets.initializer.ts

```diff
@@ -61,6 +62,11 @@ angular.module('oppia').run([
             template: componentTemplate,
             draggable: false,
             edit: function() {
+              // the following check allows editing of inline RTE components
```

Comment:
Write full sentences. Start with a capital letter, and end with a period.

Why do you need the word "inline"?

Reply:
done